test/src/XcooBee/Store/CachedDataTest.php

```diff
@@ -21,13 +21,25 @@ public function testSetStore()
         $this->assertEquals('value', Store::getInstance()->getStore('key'));
     }
 
+    public function testSetStore_StoreIsNull()
+    {
+        $store = new Store();
+        $this->assertEquals(null, $store->getStore('key'));
```

Comment:
We are calling `getStore` but function name is `testSetStore_...`. Shouldn't we be making a call to `setStore`? I'm confused.